bin/stencil-download.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+
+require('colors');
+const apiHost = 'https://api.bigcommerce.com';
+const dotStencilFilePath = './.stencil';
+const options = { dotStencilFilePath };
+const pkg = require('../package.json');
+const Program = require('commander');
+const stencilDownload = require('../lib/stencil-download');
+const versionCheck = require('../lib/version-check');
+const themeApiClient = require('../lib/theme-api-client');
+const inquirer = require('inquirer');
+
+Program
+    .version(pkg.version)
+    .option('--host [hostname]', 'specify the api host', apiHost)
+    .option('--file [filename]', 'specify the filename to download only')
+    .option('--type [type]', 'specify the type of content to download available are web-pages, theme, web-dav')
+    .parse(process.argv);
+
+if (!versionCheck()) {
+    process.exit(2);
+}
+
+const overwriteType = Program.file ? Program.file : 'files';
+
+Object.assign(options, {
+    exclude: ['parsed', 'manifest.json'],
```

Comment:
I think this by default is great but will be a good option as well where I can say give me everything except for a particular folder.

Reply:
@junedkazi - 

I don't think there is any reason to have the `parsed` and `manifest` downloaded, so I think maybe we can leave that option and introduce an `exclude` as CLI arg. What do you think?

Reply:
Sorry if I was not clear but I was referring to what you are saying 😃

Reply:
@junedkazi - Thank you!. Added the `exclude` option to CLI